transports/quic/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-quic"
-version = "0.11.2"
+version = "0.11.3"
```

```suggestion
version = "0.11.2"
```

`v0.11.2` isn't released yet